include/zephyr/net/net_pkt.h

```diff
@@ -290,6 +290,10 @@ struct net_pkt {
 	 */
 	uint8_t priority;
 
+#if defined(CONFIG_NET_OFFLOAD)
+	struct sockaddr remote;
```

Comment:
Need to document this new field. Emphasizing, that it'd only be used with NET_OFFLOAD feature.

Reply:
Added docs.